readme.md

```diff
@@ -111,19 +111,21 @@ $ npm -v
 
 * **Question:** What gets logged? Why?
 
-* **Your Answer:**
+* **Your Answer:** 
+Hello, Node! 
+Sarah
 
 * **Question:** What is `module.exports` and what is its _type_ in JavaScript? What is `require` and what is its _type_ in JavaScript?
 
-* **Your Answer:**
+* **Your Answer:** Exported file content from profile.js and imported it to index.js. "require" is a function that does something with a string, which is a path. "module.exports" is an object.
 
 ---
 
 - [ ] We can only export one thing from files when using Node. With that said, export both your name and your birthday from the `profile.js` file.
 
 * **Question:** What are some ways you can solve this problem?
 
-* **Your Answer:**
+* **Your Answer:** Create variables for each obkect that you want to export, and create an array to export with all variables you want to export.
```

Comment:
This will work! Typically though, we'll be exporting objects. Also, you don't _have_ to create variables for everything you're exporting (although it's often best practice).